less/current/modules/navs.less

```diff
@@ -9,7 +9,7 @@
   border-bottom-color: @color-light-gray;
 }
 
-.nav-tabs.nav-border-tabs > li > a {
+.nav-tabs.nav-border-tabs > li > :is(a,button) {
```

Comment:
expanded these selectors so that the styles still apply if the dropdown toggle is a button and not a link